platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/code/highlighting/codefence/CodeFenceLanguageGuesser.kt

```diff
@@ -0,0 +1,42 @@
+// Copyright 2000-2025 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.jewel.bridge.code.highlighting.codefence
+
+import com.intellij.lang.Language
+import com.intellij.lang.LanguageUtil
+import org.jetbrains.annotations.ApiStatus
+
+// Copied from org.intellij.plugins.markdown.injection.aliases.CodeFenceLanguageGuesser
+@ApiStatus.Internal
```

Comment:
We don't need the annotation if it's already internal (will also trigger Detekt if you don't pair it with Jewel's annotation)